### PR TITLE
Testing potential fix for broken 2.3.0 release of homebrew

### PR DESCRIPTION
**Pull request by filipVisko**
Temporarily locking buildkite-agent client on macOS until Homebrew is fixed.

### CODE REVIEW

File: tasks/install-on-Darwin.yml [hunk 1]
```diff
@@ -7,11 +7,18 @@
...
+  args:
+    chdir: /usr/local/Homebrew
+
 - name: Install buildkite-agent
   homebrew:
     name: buildkite-agent
-    state: latest
-    update_homebrew: yes
+    update_homebrew: no
+  environment:
```
Comment (petemounce) [inline tasks/install-on-Darwin.yml:19]: Please do one or the other, not both? It's confusing, and we won't know which one works.

File: tasks/install-on-Darwin.yml [hunk 2]
```diff
@@ -29,3 +36,12 @@
   when: buildkite_agent_load_bash_profile == True
   notify:
     - "Restart Buildkite Agent and Register for Login"
+
+- name: Reverting homebrew state back to HEAD
+  command: git checkout master
```
Comment (petemounce) [inline tasks/install-on-Darwin.yml]: This isn't the branch that homebrew checks out by default.

File: tasks/install-on-Darwin.yml [hunk 3]
```diff
@@ -29,3 +36,12 @@
...
+  args:
+    chdir: /usr/local/Homebrew
+
+- name: Refresh homebrew to original state
+  command: /usr/local/bin/brew update
```
Comment (petemounce) [inline tasks/install-on-Darwin.yml:45]: If we do this, don't do the `git checkout ...` immediately preceding it?

Comment (filipVisko) [reply tasks/install-on-Darwin.yml:45]: `brew update` does not check out the right branch so both steps are needed. I'm running brew update to validate that brew is working again.

**Comment by ca-johnson** [comment]
BK have fixed this now right?

**Pull request closed without merging**